pkg/controller/quota/quota.go

```diff
@@ -74,7 +87,10 @@ func EnsureQuotaRequest(req router.Request, resp router.Response) error {
 	// Create the quota request object and give calculate the standard numeric values
 	name, namespace, app := appInstance.Name, appInstance.Namespace, appInstance.Status.AppSpec
 	quotaRequest := &adminv1.QuotaRequestInstance{
-		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: namespace},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name, Namespace: namespace,
+			Annotations: map[string]string{labels.QuotaAppGeneration: strconv.Itoa(int(appInstance.Generation))},
```

Comment:
nit: to avoid casting
```suggestion
			Annotations: map[string]string{labels.QuotaAppGeneration: strconv.FormatInt(appInstance.Generation, 10)},
```